samples/sn/src/styles.css

```diff
@@ -1,94 +1,94 @@
-@import "~@angular/material/prebuilt-themes/indigo-pink.css";
-@import "~bootstrap/dist/css/bootstrap.min.css";
-
-.main {
-  margin-top: 56px; /* height of navbar */
-  padding: 15px 40px;
-  background-color: #f6f6ef; /* hacker news bg color */
-  min-height: 100vh;
-}
-
-.inline-block {
-  display: inline-block;
-}
-
-.score {
-  margin-right: 2px;
-}
-
-.inline-flex {
-  display: inline-flex;
-}
-
-.subtext {
-  color: grey;
-  font-size: 9pt; /* hacker news subtext font size */
-  display: inline-flex;
-}
-
-.sep {
-  border-left: 1px solid gray;
-  width: 1px;
-  height: 12px;
-  margin-top: 3px;
-  margin-left: 4px;
-  margin-right: 4px;
-}
-
-.black-sep {
-  border-left: 1px solid black;
-  width: 1px;
-  height: 15px;
-  margin-left: 4px;
-  margin-right: 4px;
-}
-
-.black-link a,
-.black-link a:hover {
-  color: black;
-  margin-left: 5px;
-  margin-right: 5px;
-}
-
-.navbar-brand a {
-  margin-left: 0px !important;
-}
-
-.mat-icon-button .mat-button-wrapper > * {
-  vertical-align: inherit !important;
-}
-
-.write-comment-container {
-  margin: 20px;
-  margin-top: 0px;
-  margin-left: 45px;
-}
-
-.write-comment-container textarea {
-  width: 300px !important;
-  height: 4rem;
-}
-
-.write-comment-container .mat-form-field, .mat-form-field-wrapper {
-  display: block;
-  width: 300px;
-}
-
-.authentication-show-user-username {
-  display: inline;
-}
-
-.dvlink, .dvlink:hover {
-  color: gray;
-}
-
-h1 {
-  font-size: 1.5rem;
-}
-
-.list-group-item {
-  background-color: initial;
-  padding: 0;
-  border: none;
-  border-bottom: 1px solid rgba(0, 0, 0, 0.125);
-}
+/*@import "~@angular/material/prebuilt-themes/indigo-pink.css";*/
```

Comment:
If you are not using this code, you can remove the file. Before you do that though, please check that the new SN looks exactly like the old one.

Reply:
resolved the new commits!